Release v1.5.0

dragos-dobre opened the pull request.

Commit:
updated Insuree Enquiry & Families sections (en only)

edarchis committed:
Update insuree page docs

edarchis committed:
Various general fixes

* fixed markdown in README.md
* updated conf.py for the latest version of sphinx (css)
* reference conflict between location and hf_claim
* Various typos

edarchis committed:
Insuree page translations

Commit:
translated families

dragos-dobre committed:
Merge pull request #32 from openimis/master

aligned with master

delcroip committed:
rev 1.5.0

delcroip committed:
title issue

delcroip committed:
vers 1.5.0 on main page

delcroip committed:
missing cap

delcroip commented on make.bat:
```diff
@@ -5,7 +5,7 @@ pushd %~dp0
 
 if "%SPHINXBUILD%" == "" (
-	set SPHINXBUILD=sphinx-build
+	set SPHINXBUILD="C:\Users\delcpa\AppData\Roaming\Python\Python38\Scripts\sphinx-build.exe"
```
this shouldn't be pushed

edarchis committed:
Adapted Policies page & extracted date_picker

edarchis committed:
Contributions module update

delcroip committed:
add mobile manual

delcroip committed:
add mobile

edarchis committed:
Medical services and items modular rewrite

Update the docs for medical items and services. No translation yet as it yields some compilation errors.

delcroip committed:
formal sector

delcroip committed:
fixes

qgerome committed:
Merge pull request #34 from openimis/resync_insuree_family_contrib

Resync insuree family contrib
